packages/server/src/fhir/operations/export.test.ts

```diff
@@ -99,211 +99,6 @@ describe('System export', () => {
     expect(JSON.parse(resourceJSON[0])?.subject?.reference).toEqual(`Patient/${res1.body.id}`);
   });
 
-  test('Parameters', async () => {
```

Comment:
These tests using `_since` parameter is redundant and doesn't provide additional test coverage.
`since` parameter is also tested in https://github.com/medplum/medplum/blob/e8bff4347550771ffa7ee1e3aa1d4ba887d17452/packages/server/src/fhir/operations/groupexport.test.ts#L128